homeassistant/components/uptimerobot/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
```

Comment:
```py
    @property
    def is_on(self) -> bool:
        """Return True if the entity is on."""
        return bool(self.monitor.status != 0)
```
it's always a bool, no need to `bool()` it

Reply:
Unfortunately not, without the `bool`:

```
homeassistant/components/uptimerobot/switch.py:53: error: Returning Any from function declared to return "bool"  [no-any-return]
Found 1 error in 1 file (checked 1 source file)
```

Reply:
oh interesting, I am wondering why this is the case